src/utils/DockerUtil.js

```diff
@@ -263,6 +271,68 @@ var DockerUtil = {
     });
   },
 
+  fetchAllNetworks () {
+    this.client.listNetworks((err, networks) => {
+      if (err) {
+        networkActions.error(err)
+      } else {
+        networks = networks.filter(n => n.Name !== 'host').sort((n1, n2) => {
```

Comment:
Why is `host` network filtered out here? Granted for 99% of the cases it's best to avoid it